[BUG] Defining 'Ws' or 'Gs' in custom circuits fails

**Describe the bug**
Trying to define a custom circuit with 'Ws' or 'Gs' raises an error that the number of initial guesses doesn't match the circuit length. This seems to come from `extract_circuit_elements` incorrectly removing the `s` at the end of each element name.

**To Reproduce**
```
import sys
sys.path.append('../../../')

from impedance.models.circuits import CustomCircuit
```
```
circ = 'R_0-Gs_1-Ws_1'
inits =[10, 100, .1, .3, 50, 1]
sim = CustomCircuit(circuit=circ, initial_guess=inits)

>>AssertionError: The number of initial guesses + the number of constants (6) needs to be equal to the circuit length (4)
```

**Expected behavior**
The circuit should be successfully created with 6 initial guess parameters. The number of parameters for each element are: R_0 = 1, Gs_1 = 3, Ws_1 = 2.

**Additional context**
Running the following shows that `Gs` and `Ws` are incorrectly identified as `G` and `W`
```
from impedance.models.circuits.fitting import extract_circuit_elements
extract_circuit_elements(circ)

>>['R_0', 'G_1', 'W_1']
```

--- a/impedance/models/circuits/fitting.py
+++ b/impedance/models/circuits/fitting.py
@@ -241,7 +241,7 @@ def buildCircuit(circuit, frequencies, *parameters,
 
 def extract_circuit_elements(circuit):
     extracted_elements = []
-    p_string = [x for x in circuit if x not in 'ps(),-/']
+    p_string = [x for x in circuit if x not in 'p(),-']
     extracted_elements = []
     current_element = []
     length = len(p_string)

